props/saber_fett263_buttons.h

```diff
@@ -2617,14 +2627,18 @@ SaberFett263Buttons() : PropBase() {}
       case STYLE_OPTION_ARG:
 	menu_type_ = MENU_STYLE_OPTION;
 	arg_revert_ = strtol (argspace, NULL, 0);
+        sound_library_.SayOption();
+        sound_library_.SayNumber(calc_, SAY_WHOLE);
 	break;
       case IGNITION_OPTION_ARG:
 	menu_type_ = MENU_IGNITION_OPTION;
 	arg_revert_ = strtol (argspace, NULL, 0);
+        sound_library_.SayOption();
+        sound_library_.SayNumber(calc_, SAY_WHOLE);
 	break;
       case IGNITION_TIME_ARG:
 	menu_type_ = MENU_IGNITION_TIME;
-	arg_revert_ = strtol (argspace, NULL, 0);
+	arg_revert_ = strtol (argspace, NULL, 0);		      
```

Comment:
Looks like you just added some whitespace at the end of the line here. Please fix.

Reply:
fixed